cloud/ovirt/ovirt_clusters.py

```diff
@@ -0,0 +1,219 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+#
+# Copyright (c) 2016 Red Hat, Inc.
+#
+# This file is part of Ansible
+#
+# Ansible is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# Ansible is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with Ansible.  If not, see <http://www.gnu.org/licenses/>.
+#
+
+try:
+    import ovirtsdk4.types as otypes
```

Comment:
We check the SDK and raise friendly error [here](https://github.com/ansible/ansible-modules-extras/pull/3138/files#diff-cb60a765df80bc6d3e43ca8b069a7e2fR193)
which has [following](https://github.com/ansible/ansible/blob/devel/lib/ansible/module_utils/ovirt.py#L47) implementation.
We use it so we can manage the error message and version check at one place.